tests/Metalama.Framework.Tests.Integration/Tests/Aspects/Introductions/Namespaces/Recursive.t.cs

```diff
@@ -0,0 +1,22 @@
+// --- Recursive.cs ---
+// Warning CS8618 on `Field`: `Non-nullable field 'Field' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.`
+[IntroductionAttribute]
+public class TargetType
+{
+  private global::Outer.Middle.Inner.Test Field;
+}
+// --- Outer.cs ---
+namespace Outer
```

Comment:
Namespaces should not require a syntax tree - they are symbolic introductions, without underlying syntax.